docs/mkdocs/docs/faq.md

```diff
@@ -40,24 +40,55 @@ of both batch and streaming data.
 
 Bindings are currently only available for Python. 
 
-###*How can I get started using ArcticDB?*
+### *What is the best practice for saving Data to ArcticDB?*
+
+Users are encouraged to store data in its original vendor format initially, then optimize storage based on their specific use cases, which may include maintaining multiple versions of the data to support different analysis needs. In ArcticDB, best practices for data storage involve considering the balance between read and update performance, data layout’s impact on index performance, and auditability against original data sources. It’s recommended to store data in a format that facilitates efficient querying. 
+
+A full guide can be found [here](tutorials/data_organisation.md)!
+
+### *What are the Limitations of ArcticDB being client-side?*
+
+The serverless nature of ArcticDB provides excellent performance, making it ideal for data science applications where speed and efficiency are key. Although it may not be ideal for scenarios that demand strong transactionality and concurrent updates, ArcticDB excels in complex, multi-tenant environments where its attributes align closely with those of popular NoSQL databases such as MongoDB and Cassandra, offering flexibility and scalability. 
+
+### *What storage options does ArcticDB support?*
+
+ArcticDB offers compatibility with a wide array of storage choices, both on-premises and in the cloud, to guarantee adaptability and efficiency across different settings. It is verified to work with multiple storage systems such as AWS S3, Azure Blob, lmdb, In-memory, Ceph, MinIO, Pure Storage S3, Scality S3, and VAST Data S3, with plans to support additional options soon. 
+
+
+### *What are the trade offs with ArcticDB Versioning?* 
+
+ArcticDB offers robust versioning capabilities, allowing for point-in-time analysis and efficient data updates, including daily appends and historical corrections, making it ideal for research datasets. The database efficiently stores only the differences between versions, conserving storage space. Frequent, small updates can lead to fragmentation and slower reads, but ArcticDB's enterprise features can mitigate this through background compaction and data pruning.  
+
+More information can be found [here](tutorials/data_organisation.md)!
+
+### *Does ArcticDB support Authorization?*  
+
+ArcticDB itself does not inherently support authorization at the moment as it operates fully on the client side. However, it is designed to work with first and third-party authentication solutions for managing data access. ArcticDB data can be effectively managed through the underlying storage system, such as S3 and Blob. Users can utilize storage permissions to create personal buckets with restricted write access or team buckets with broader read/write permissions. Permissions can also be set on individual ArcticDB libraries by configuring the access rights for each library's corresponding path.
```

Comment:
Remove "as it operates fully on the client side". It is possible to support auth on the client side and we plan to add features for this in future.